src/test/java/UnitTests/JsonParserTests/JsonParserReadTests.java

```diff
@@ -0,0 +1,76 @@
+package UnitTests.JsonParserTests;
+
+import com.google.gson.JsonSyntaxException;
+import org.junit.gen5.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.testng.Assert;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import parser.Parser;
+import shop.Cart;
+import java.io.File;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class JsonParserReadTests {
+
+    @ParameterizedTest
+    @ValueSource(strings = { "sample-cart.json" })
+    void readFromFileTest(String fileName) {
+        Parser parser = new JsonParser();
+        Cart cart;
+
+        cart = parser.readFromFile(new File("./src/test/java/UnitTests/TestData/" + fileName));
+        cart.showItems();
+
+        Assert.assertNotNull(cart);
```

Comment:
Please add error message as a second param here, and it's good approach to add such messages for all assertions